commit 41a0197881ec8f9fef02279e78446196d2432e20
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-04-23T14:38:28-07:00 (6 months ago)
    
    TRITON-1628 migration provision test failing with "invalid networks" when no fabrics available

diff --git a/test/vms.migrate.test.js b/test/vms.migrate.test.js
index 75f8813..c7d4347 100644
--- a/test/vms.migrate.test.js
+++ b/test/vms.migrate.test.js
@@ -40,7 +40,7 @@ var testMigration = require('./lib/migration');
 /* Globals */
 
 var ADMIN_USER_UUID = common.config.ufdsAdminUuid;
-var PROVISION_NETWORKS = [];
+var PROVISION_NETWORKS;
 var client;
 
 var configurations = [
@@ -188,7 +188,10 @@ exports.config_setup = function configSetup(t) {
             assert.string(cfg.packageName, 'cfg.packageName');
             assert.object(cfg.vm, 'cfg.vm');
 
-            cfg.vm.networks = [PROVISION_NETWORKS];
+            cfg.vm.networks = [];
+            if (PROVISION_NETWORKS) {
+                cfg.vm.networks.push(PROVISION_NETWORKS);
+            }
 
             vasync.pipeline({funcs: [
                 function loadPackages(_, next) {
